nixos/doc/manual/release-notes/rl-1909.xml

```diff
@@ -403,6 +403,26 @@
      installer after creating <literal>/var/lib/nextcloud</literal>.
     </para>
    </listitem>
+   <listitem>
+    <para>
+      The <literal>/etc/sysctl.d/nixos.conf</literal> file containing all the options set via
+      <link linkend="opt-boot.kernel.sysctl">boot.kernel.sysctl</link> was moved to
+      <literal>/etc/sysctl.d/60-nixos.conf</literal>, as <literal>sysctl.d(5)</literal>
+      recommends prefixing all filenames in <literal>/etc/sysctl.d</literal> with a
+      two-digit number and a dash to simplify the ordering of the files.
+    </para>
+   </listitem>
+   <listitem>
+     <para>
+       We now install the sysctl snippets shipped with systemd. Apart from some loose reverse path
+       filtering and source routing filtering, this enables <literal>fq_codel</literal> as a packet
```

Comment:
"some loose reverse path filtering and source routing filtering" sounds important. Can you go in to more detail about what changed from (old) to (new), and its impact?

Reply:
see below comment.